gapis/trace/android/mali/validate.go

```diff
@@ -85,6 +97,18 @@ func (v *MaliValidator) GetCounters() []validate.GpuCounter {
 		strings.HasSuffix(gpuName, "G78AE") {
 		return jmCounters
 	} else {
-		return csfCounters
+		/*
+		 * Driver versions reported via Vulkan have the following format:
+		 * | 31 .. 29 | 28 .. 22 | 21 .. 12 | 11 .. 0 |
+		 * | variant  |  major   |  minor   |  patch  |
+		 */
```

Comment:
go style prefers `//` comments, even for multi-line, for some reason.

Reply:
Thanks, I'll get it changed. It's not something `go fmt` flagged so thought it'd be fine.

Reply:
Thank you! gofmt doesn't lint/check style. Here's the style guide: https://go.dev/doc/effective_go#commentary, :man_shrugging: